src/Controller/Cliente/ClienteController.php

```diff
@@ -26,4 +30,97 @@ public function clientes(ClienteRepository $clienteRepository, SerializerInterfa
             );
         }
     }
+    #[Route("/clientes", name: "app_clientes", methods: ['GET'])]
+    public function clientes(ClienteRepository $clienteRepository): Response
+    {
+        $clientes = $clienteRepository->findAll();
+        return $this->render("cliente/clientes.html.twig",["clientes" => $clientes]);
+    }
+
+    #[Route("/clientes/adicionar", name: "app_adicionarCliente", methods: ['GET'])]
```

Comment:
deixa sem, ses executar um debug router vai ficar app_app...

```suggestion
    #[Route("/clientes/adicionar", name: "adicionarCliente", methods: ['GET'])]
```